libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
Sensu and Logstash are pretty much the only ones that I can think of. Anyhow, this is not C and namespacing of functions is a solved problem. You can always invoke a function with a module name specified.